src/containers/MyLists/index.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import {
+  Layout, Row, Col, Card, Typography, Modal, Pagination
+} from 'antd'
+import { PlusCircleOutlined, DeleteOutlined } from '@ant-design/icons'
+import { range } from 'lodash'
+
+import Header from '../../components/Header'
+// import CreateListModal from '../CreateListModal'
+
+const showModal = () => {
+  Modal.confirm({
+    title: 'Do you want to delete list?',
+    onOk() {},
+    onCancel() {}
+  })
+}
+
+class Lists extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { modalVisible: false }
+
+    this.showModal = () => {
+      this.setState({ modalVisible: true })
+    }
+
+    this.hideModal = () => {
+      this.setState({ modalVisible: false })
+    }
+  }
```

Comment:
you can do this ouside the constructor like:
```
    hideModal = () => {
      this.setState({ modalVisible: false })
    }
```

Reply:
pls, don't review components outside the Login feature